src/SharpSvn/SharpSvnStrings.h

```diff
@@ -1,854 +1,854 @@
-        return ResourceManager->GetString(L"RevisionTypeMustBeHeadDateOrSpecific", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::SessionAlreadyOpen::get() {
-        return ResourceManager->GetString(L"SessionAlreadyOpen", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::SvnAuthManagedPlaceholder::get() {
-        return ResourceManager->GetString(L"SvnAuthManagedPlaceholder", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::SvnClientOperationInProgress::get() {
-        return ResourceManager->GetString(L"SvnClientOperationInProgress", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::SvnMalfunctionPrefix::get() {
-        return ResourceManager->GetString(L"SvnMalfunctionPrefix", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::TargetMustContainExplicitRevision::get() {
-        return ResourceManager->GetString(L"TargetMustContainExplicitRevision", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::TheTargetIsNotAValidPathTarget::get() {
-        return ResourceManager->GetString(L"TheTargetIsNotAValidPathTarget", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::TheTargetIsNotAValidUriOrPathTarget::get() {
-        return ResourceManager->GetString(L"TheTargetIsNotAValidUriOrPathTarget", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::TheTargetIsNotAValidUriTarget::get() {
-        return ResourceManager->GetString(L"TheTargetIsNotAValidUriTarget", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::UriIsNotAbsolute::get() {
-        return ResourceManager->GetString(L"UriIsNotAbsolute", resourceCulture);
-    }
-    
-    inline System::String^  SharpSvnStrings::VerifyEnumFailed::get() {
-        return ResourceManager->GetString(L"VerifyEnumFailed", resourceCulture);
-    }
-}
+﻿#pragma once
+
+#using <mscorlib.dll>
+#using <System.dll>
+#using <System.dll>
+
+using namespace System::Security::Permissions;
+[assembly:SecurityPermissionAttribute(SecurityAction::RequestMinimum, SkipVerification=false)];
+namespace SharpSvn {
+    using namespace System;
+    using namespace System;
+    ref class SharpSvnStrings;
+    
+    
+    /// <summary>
+    ///   Eine stark typisierte Ressourcenklasse zum Suchen von lokalisierten Zeichenfolgen usw.
+    /// </summary>
+    // Diese Klasse wurde von der StronglyTypedResourceBuilder automatisch generiert
+    // -Klasse über ein Tool wie ResGen oder Visual Studio automatisch generiert.
+    // Um einen Member hinzuzufügen oder zu entfernen, bearbeiten Sie die .ResX-Datei und führen dann ResGen
+    // mit der /str-Option erneut aus, oder Sie erstellen Ihr VS-Projekt neu.
+    [System::CodeDom::Compiler::GeneratedCodeAttribute(L"System.Resources.Tools.StronglyTypedResourceBuilder", L"4.0.0.0"), 

```

Comment:
Nice... generated the German texts as default ;-)

Reply:
Reverted changes to the file and commited old version